### PR TITLE
[ignore] README additional not on nd connection plugin usage

### PR DESCRIPTION
**Pull request by akinross**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ This collection has been tested and supports MSO 2.1+.
 Modules supporting new features introduced in MSO API in specific MSO versions might not be supported in earlier MSO releases.
 
 *Note: This collection is not compatible with versions of Ansible before v2.8.*
+*Note: The Nexus Dashboard (ND) HTTPAPI connection plugin should be used when Cisco ACI Multi-Site is installed on Nexus Dashboard.*
```
Comment (shrsr) [inline README.md:10]: Would it be worth mentioning that all MSO versions post 3.2 need to use the 
ND plugin?

Comment (lhercot) [inline README.md:10]:
```suggestion
*Note: The Nexus Dashboard (ND) HTTPAPI connection plugin should be used when Cisco ACI Multi-Site is installed on Nexus Dashboard (v3.2+) or when using this collection with Nexus Dashboard Orchestrator (v3.6+).*
```

File: README.md [hunk 2]
```diff
@@ -22,7 +23,12 @@ Install the collection
 ansible-galaxy collection install cisco.mso
 ```
 
-## Use
+Install the ND collection when Cisco ACI Multi-Site is installed on Nexus Dashboard
```
Comment (lhercot) [inline README.md:26]:
```suggestion
Install the ND collection when Cisco ACI Multi-Site is installed on Nexus Dashboard (v3.2+) or when using this collection with Nexus Dashboard Orchestrator (v3.6+)
```

**Review by lhercot — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -63,6 +69,7 @@ You can also use the Nexus Dashboard (ND) connection plugin available in the [ci
 ```yaml
```
Comment (lhercot) [inline README.md:69]: Also add verbiage in this paragraph

**Review by lhercot — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (akinross) [reply README.md:69]: done in line 68

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**